src/components/Activities/Activities.tsx

```diff
@@ -0,0 +1,73 @@
+// eslint-disable-next-line @typescript-eslint/ban-ts-comment
+// @ts-ignore
+
+export default function Activities() {
+  const list = [
```

Comment:
Eventually we're going to need this component to accept props. Can you please move this activities array out to `App` and then pass it into `ActivitiesList` as a prop called `activities`?